Chap_API_Fabric.tex

```diff
@@ -374,13 +374,13 @@ \subsection{Fabric Coordinate Views}
 
```

Comment:
Now that we have these definitions hard coded, do we want to get rid of the \copySignature code on line 366?  Seems unnecessary now?